src/log-store/src/kafka/client_manager.rs

```diff
@@ -67,7 +67,7 @@ pub(crate) struct ClientManager {
     client_factory: RsKafkaClient,
     /// A pool maintaining a collection of clients.
     /// Key: a topic. Value: the associated client of the topic.
-    client_pool: DashMap<Topic, Client>,
+    client_pool: Mutex<HashMap<Topic, Client>>,
```

Comment:
I think maybe we can use `RwLock` for this case, it's much more reading than writing.

Reply:
Replaced Mutex with RwLock